docs/en_US/reference/experiment_config.rst

```diff
@@ -0,0 +1,575 @@
+
+Detailed `here <https://nni.readthedocs.io/en/stable/TrainingService/RemoteMachineMode.html>`_.
+
+platform
+--------
+
+Constant string ``"remote"``.
+
+
+machine list
+------------
+
+List of training machines.
+
+type: list of `RemoteMachineConfig`_
+
+
+reuse mode
+----------
+
+Enable reuse mode [TODO]
+
+type: bool
+
+
+RemoteMachineConfig
+===================
+
+host
+----
+
+IP or hostname (domain name) of the machine.
+
+type: ``str``
+
+
+port
+----
+
+SSH service port.
+
+type: ``int``
+
+default: 22
+
+
+user
+----
+
+Login user name.
+
+type: ``str``
+
+
+password
+--------
+
+Login password.
+
+type: ``Optional[str]``
+
+If not specified, `ssh key file`_ will be used instead.
+
+
+ssh key file
+------------
+
+`Path`_ to ssh key file (identity file).
+
+type: ``str``
+
+default: ``"~/.ssh/id_rsa"``
+
+Only used when `password`_ is not specified.
+
+
+ssh passphrase
+--------------
+
+Passphrase of SSH identity file.
+
+type: ``Optional[str]``
+
+
+use active gpu
+--------------
+
+Specify whether NNI should submit trials to GPUs occupied by other tasks.
+
+type: ``bool``
+
+
+max trial number per gpu
+------------------------
+
+Specify how many trials can share one GPU.
+
+type: ``int``
+
+default: ``1``
+
+
+gpu indices
+-----------
+
+Limit the GPUs visible to trial processes.
+
+type: ``Optional[Union[list[int], str]]``
+
+If `trial gpu number`_ is less than the length of this value, only a subset will be visible to each trial.
+
+This will be used as ``CUDA_VISIBLE_DEVICES`` environment variable.
+
+
+trial prepare command
+---------------------
+
+Command(s) to run before launching each trial.
+
+type: ``Optional[str]``
+
+This is useful if preparing steps vary for different machines.
+
+
+OpenPaiConfig
+=============
+
+Detailed `here <https://nni.readthedocs.io/en/stable/TrainingService/PaiMode.html>`_.
+
+platform
+--------
+
+Constant string ``"openpai"``.
+
+
+host
+----
+
+Hostname of OpenPAI service.
+
+type: ``str``
+
+
+username
+--------
+
+OpenPAI user name.
+
+type: ``str``
+
+
+token
+-----
+
+OpenPAI user token.
+
+type: ``str``
+
+This can be found in your OpenPAI user settings page.
+
+
+trial cpu number
+----------------
+
+Number of CPUs used by each trial.
+
+type: ``int``
+
+default: ``1``
+
+
+trial memory size
+-----------------
+
+Memory used by each trial.
+
+type: ``str``
+
+examples: ``"1gb"``, ``"512mb"``
+
+
+docker image
+------------
+
+Name and tag of docker image to run the trials.
+
+type: ``str``
+
+default: ``"msranni/nni:latest"``
+
+
+reuse mode
+----------
+
+Enable reuse mode.
+
+type: ``bool``
+
+default: ``False``
+
+
+nni manager storage mount point
+-------------------------------
+
+`Mount point <path>`_ of storage service (typically NFS) on current machine.
+
+type: ``str``
+
+
+container storage mount point
+-----------------------------
+
+Mount point of storage service (typically NFS) in docker container.
+
+type: ``str``
+
+This must be an absolute path.
+
+
+open pai config
```

Comment:
I think openpai is a word (at least OpenPAI is).

Reply:
So it will be `openpaiConfig` and `openpai_config` in yaml and python respectively. I think it looks weird. Discuss it in meeting.